File: ApiSuvesaPos/Negocio/Modelo/cxp/compras/articulos_gastos.vb
Namespace Modelo
    Public Class articulos_gastos
        Public Property Id_ArticuloComprados As Integer
        Public Property IdCompra As Integer
        Public Property Descripcion As String
        Public Property Base As Decimal
        Public Property Monto_Flete As Decimal
        Public Property OtrosCargos As Decimal
        Public Property Costo As Decimal
        Public Property Cantidad As Decimal
        Public Property Gravado As Decimal
        Public Property Exento As Decimal
        Public Property Descuento_P As Decimal
        Public Property Descuento As Decimal
        Public Property Impuesto_P As Decimal
        Public Property Impuesto As Decimal
        Public Property Total As Decimal
        Public Property Devoluciones As Decimal
        Public Property NuevoCostoBase As Decimal
        Public Property CuentaContable As String
        Public Property CodTipoCompra As Integer
        Public Property DescTipoCompra As String
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxp/compras/articulos_gastos.cs
namespace Modelo
{
    public class articulos_gastos
    {
        public int Id_ArticuloComprados { get; set; }
        public int IdCompra { get; set; }
        public string Descripcion { get; set; }
        public decimal Base { get; set; }
        public decimal Monto_Flete { get; set; }
        public decimal OtrosCargos { get; set; }
        public decimal Costo { get; set; }
        public decimal Cantidad { get; set; }
        public decimal Gravado { get; set; }
        public decimal Exento { get; set; }
        public decimal Descuento_P { get; set; }
        public decimal Descuento { get; set; }
        public decimal Impuesto_P { get; set; }
        public decimal Impuesto { get; set; }
        public decimal Total { get; set; }
        public decimal Devoluciones { get; set; }
        public decimal NuevoCostoBase { get; set; }
        public string CuentaContable { get; set; }
        public int CodTipoCompra { get; set; }
        public string DescTipoCompra { get; set; }
    }
}
